src/test/ui/self/elision/self.rs

```diff
@@ -0,0 +1,36 @@
+// run-pass
```

Comment:
If all you have is `fn main() {}` prefer using `// check-pass` since there are no dynamic semantics to test.
(Applies to various test files in the PR)

Reply:
Addressed in https://github.com/rust-lang/rust/pull/61207/commits/a1b813b4da3ab048bfc412ce878e6266279c358d.